.github/workflows/e2e-tests.yml

```diff
@@ -47,39 +47,42 @@ jobs:
             # Function to get deployment URLs
             get_deployment_url() {
               DEPLOYMENT_URLS="${{ github.event.deployment_status.environment_url }}"
-              if [[ "$DEPLOYMENT_URLS" != *"styleguide"* ]]; then
+              if [[ "$DEPLOYMENT_URLS" != *"styleguide"* && -n "$DEPLOYMENT_URLS" ]]; then
                 echo "$DEPLOYMENT_URLS"
                 return 0
               fi
               return 1
             }
```

Comment:
I'm not sure why we're even doing all this long-polling to "find" the right url?

We don't want this GitHub action to run _at all_ if the event that was captured is for the styleguide project.

This isn't the right approach. We need to look into how the GitHub Action is triggered and make sure that _only_ happens for the webapp, _not_ for the styleguide.